README.adoc

```diff
@@ -468,6 +468,15 @@ external_services:
     service: VALUE
 ----
 
+|`JAEGER_SERVICE_PORT`
+|The Kubernetes service port for Jaeger. This configuration is ignored if `JAEGER_URL` is set. (default is `16686`)
```

Comment:
I don't think it's true that it is ignored when JAEGER_URL is set. Actually, all the proxy stuff is used whatever is set for JAEGER_URL, no?

Reply:
If the JAEGER_URL is used, the GetJaegerInfo will return that URL ignoring the Jaeger proxy.
At least in the current code.

Reply:
That's not what I read: https://github.com/kiali/kiali/blob/master/handlers/jaeger.go#L45-L48
It's still going to pass through proxy, overriding any URL. Or am I missing something?

Reply:
Yeah, true there is another potential problem in that side.
Current code proxy is used in any case.